packages/atoms/src/SearchInput.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { TextInput, Pressable, View } from 'react-native';
+import { Box } from './Box';
+import { Icon } from './Icon/Icon';
+
+const MAX_LENGTH = 255;
+
+type InputProps = {
+    value: string;
+    onChange: (value: string) => void;
+    placeholder: string;
+    isClearable?: boolean;
+    isDisabled?: boolean;
+};
+
+type InputStyleProps = {
+    isFocused: boolean;
+};
+
+const inputStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    fontSize: 16,
+    color: utils.colors.gray700,
+    height: '100%',
+    marginLeft: utils.spacings.md,
+}));
+
+const clearIconStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.gray500,
+    height: 20,
+    width: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: utils.borders.radii.round,
+}));
+
+const inputWrapperStyle = prepareNativeStyle<InputStyleProps>((utils, { isFocused }) => ({
```

Comment:
I would prefer to have InputWrapper as standalone component, because we will need exact same wrapper in Input a probably Select too

Reply:
[fd66c65](https://github.com/trezor/trezor-suite/pull/5433/commits/fd66c6563cb2faf0fc3e106e2fd2f34166bbec9b)